plugin/src/directive_comments.rs

```diff
@@ -49,7 +49,7 @@ impl<'a> FromFileRunContext<'a> for DirectiveComments<'a> {
                         for (id, string_config) in
                             config_comment_parser::parse_string_config(directive_value, comment)
                         {
-                            let normalized_value = match serde_json::from_str::<globals::Visibility>(
+                            let normalized_value = match serde_yaml::from_str::<globals::Visibility>(
```

Comment:
I ran into this failing on the test case with something like `/* globals parseInt:off */`, so since the parsed "value" for these globals doesn't apparently necessarily have quotes around out (or is `true`/`false`) it seemed like the easiest fix was to interpret it as YAML instead of as JSON (and I don't see any existing tests failing as a result of this change)